### ISSUE TITLE
Unable to parse >=

### ISSUE THREAD

**Issue by louderspace** [brendan-duncan/wgsl_reflect#54]
Before investing in using wgsl_reflect, I tried the test page:

https://brendan-duncan.github.io/wgsl_reflect/example.html

.. with my shader and found it is unable to parse this function:

[highlightOverRangeColors.txt](https://github.com/user-attachments/files/16607313/highlightOverRangeColors.txt)

Specifically these lines with >=:

var cEdgeBool : vec3<bool> = c3 >= vec3<f32>(1.0);
var modVec : vec2<f32> = floor(pos / checkerSize) % 2;

**Comment by brendan-duncan** [comment]
What problem are you seeing?

**Comment by louderspace** [comment]
This is strange.  On my work laptop, I simply pasted the code between the comments "//######" into your test page:

https://brendan-duncan.github.io/wgsl_reflect/example.html

And it produced errors.  On my home MBP, it's not happening.  Let me try again on my other laptop and I'll paste the error.

**Comment by brendan-duncan** [comment]
Thanks, I can repo it now. Very strange, but I'll get it sorted out.

**Comment by brendan-duncan** [comment]
Figured out the issue, just have to find a solution. I have code in the scanner that tries to disambiguate `>=` between being a `greater-equal` or `closing of a template followed by equals`, as in   `a >= b` from `vec2<bool> = b`.

Your code found the hole in my logic. Back to the drawing board.

**Comment by brendan-duncan** [comment]
The function is parsing now.

**Comment by louderspace** [comment]
I also verified this issue has been resolved, thank you!
